client/my-sites/site-settings/placeholder.jsx

```diff
@@ -0,0 +1,76 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { localize } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import Card from 'components/card';
+import Main from 'components/main';
+import NavItem from 'components/section-nav/item';
+import NavTabs from 'components/section-nav/tabs';
+import SectionHeader from 'components/section-header';
+import SectionNav from 'components/section-nav';
+
+const Placeholder = ( { translate } ) => {
+	const header = (
+		<div className="site-settings__placeholder-item">
+			General Settings
```

Comment:
Thanks for the feedback! ✋ 

I think we're good like that. In that case, it does not matter if we'll have "General Settings", "Example Content" or "Meh, meh meh meh" there, right 😉  There's not a real standard with what content we render there, and that's justified by the fact that this content is not even visible.

The only reason I have the translated values for the nav menu placeholder is because this way the items placeholders will be as realistic as possible in terms of width. And I don't think the rest of the placeholders need to be realistic in terms of width. The only so-called-limitation was that title should not be too long, so it would look like a title indeed.